transforms/code/semantic_profiler/ray/test-data/expected/metadata.json

```diff
@@ -0,0 +1,46 @@
+{
+    "pipeline": "pipeline_id",
+    "job details": {
+        "job category": "preprocessing",
+        "job name": "NOOP",
```

Comment:
this file needs to be regenerated since it seems to have been copied from the NOOP project.